lib/accessly/permission/grant.rb

```diff
@@ -33,13 +33,13 @@ def initialize(actor)
       #     # Allow the user access to posts for action id 3 on a segment
       #     Accessly::Permission::Grant.new(user).on_segment(1).grant!(3, "posts")
       #
-      # @overload grant!(action_id, object_type, object_id)
-      #   Allow permission on an ActiveRecord object.
+      # @overload grant!(action_id, namespace, namespace_id)
+      #   Allow permission on a namespaced object.
       #   A grant is universally unique and is enforced at the database level.
       #
       #   @param action_id [Integer] The action to grant for the object
-      #   @param object_type [ActiveRecord::Base] The ActiveRecord model that receives a permission grant.
-      #   @param object_id [Integer] The id of the ActiveRecord object which receives a permission grant
+      #   @param namespace [ActiveRecord::Base] The ActiveRecord model that receives a permission grant.
+      #   @param namespace_id [Integer] The id of the ActiveRecord object which receives a permission grant
```

Comment:
I understand the change from `object_type` to `namespace`, but perhaps it makes more sense to keep `object_id` as `object_id`?

My thinking here is that `object_id` represents an actual object and isn't necessarily tied to the namespace. And `namespace_id` to me sounds like an ID we gave the namespace itself.

@ehourigan and @rreinhardt9, what do you think?

Reply:
Additionally, we have the `PermittedActionOnObject` model, which suggests to me that the column name should follow the table and model name.

Reply:
Are we complicating things with this change? (I kind of feel this way)

Reply:
With my suggestion or with this whole PR?

Reply:
Sorry, the whole pr

Reply:
Gotcha. I'm not terribly convinced this change has to happen, but I can see why we might want it to happen. If it feels unnecessary, I'm happy to cancel the whole thing.

Even if we put it off, it doesn't change the API at all (but we will have to be careful if we try to change column names in the future).

Reply:
I could definitly see `object_id` being just fine as object id :+1:. In fact, it makes more sense to me that way because of the reasons you named. I'm also fine either way on changing/ not changing object_type. Really, I feel like the think that it's referring to is more like "policy" or "policy_name" more than a namespace... but 🤷‍♂️  naming things is hard 😆

Reply:
Welp, shall we consider this to be "too hard to name" and leave it as is and close this PR?